pkg/util/push/push_test.go

```diff
@@ -20,15 +21,17 @@ import (
 func TestHandler_remoteWrite(t *testing.T) {
 	req := createRequest(t, createPrometheusRemoteWriteProtobuf(t))
 	resp := httptest.NewRecorder()
-	handler := Handler(distributor.Config{MaxRecvMsgSize: 100000}, verifyWriteRequestHandler(t, client.API))
+	var sourceIPs *middleware.SourceIPExtractor
+	handler := Handler(distributor.Config{MaxRecvMsgSize: 100000}, sourceIPs, verifyWriteRequestHandler(t, client.API))
```

Comment:
Can be simplified, if your intention is to not pass the IP extractor.

```suggestion
	handler := Handler(distributor.Config{MaxRecvMsgSize: 100000}, nil, verifyWriteRequestHandler(t, client.API))
```

Reply:
Good catch. The variable declaration has been removed.